style.css

```diff
@@ -3005,7 +3005,7 @@ ul.wp-block-gallery:not(.alignwide):not(.alignfull):not(.alignright):not(.alignl
 	display: block;
 	font-size: 6.2rem;
 	font-weight: 500;
-	line-height: 1.2em;
+	line-height: 12px;
```

Comment:
This should be changed to `line-height: 1.2`, to make sure the relative line height is retained when the font size is increased.